Sources/PrimerSDK/Classes/User Interface/Apple Pay/ApplePayViewController+PassKit.swift

```diff
@@ -1,112 +1,113 @@
-#if canImport(UIKit)
```

Comment:
[Q]
What's the plan with this file?

Reply:
No plan actually, this was Carl's implementation that I commented out so I could be having a look on specific decisions that he wanted to take. Removing it.